src/components/Text.tsx

```diff
@@ -243,3 +243,23 @@ const proofSectionTitle = classnames(
 export function ProofSectionTitle({ children }: ChildrenProp) {
   return <p className={proofSectionTitle}>{children}</p>
 }
+
+const tinyText = (color: 'base' | 'primary' | 'error') =>
+  classnames(
+    textColor(
+      color === 'error'
+        ? 'text-error'
+        : color === 'primary'
+        ? 'text-primary-semi-dimmed'
+        : 'text-formal-accent'
+    ),
+    fontFamily('font-primary'),
+    fontSize('text-xs'),
+    lineHeight('leading-3')
+  )
+export function TinyText({
```

Comment:
In my opinion, `tinyText` is not self-explanatory, why can't we just make a tiny version of any other texts? Maybe it's more like a `hintText`?

Reply:
I do agree with @smyzin, it's better to try to create unified `TinyText`, than many different small texts